nix/dev.nix

```diff
@@ -2,11 +2,7 @@
 pkgs.recurseIntoAttrs (rec {
   # Packages which are useful during development, but we don't depend upon directly to build our stuff
   packages = pkgs.recurseIntoAttrs {
-    # See comment on the definition about it not working
-    #cabal-install = haskell.extraPackages.cabal-install.components.exes.cabal;
-    stylish-haskell = haskell.extraPackages.stylish-haskell.components.exes.stylish-haskell;
-    hlint = haskell.extraPackages.hlint.components.exes.hlint;
-    haskell-language-server = haskell.extraPackages.haskell-language-server.components.exes.haskell-language-server;
```

Comment:
This is different to what was there before: we were specifically selecting the right executable component. Does just using the whole package work? I thought I'd tried that and it didn't work...

Reply:
The exe selection is done in nix/haskell-extra.nix (mostly by the `haskell-nix.tools` function).

Reply:
Okay, then this is covered by the discussion about using `haskell-nix.tools`.